### ISSUE TITLE
MultiActionButton - Cannot go through options with NVDA

### ISSUE THREAD

**Issue by gusch** [Sage/carbon#6244]
### Current behaviour

With NVDA active, focus on a MultiActionButton.

I can't open the options list with enter or down arrow.

When the list is opened, i can't navigate through the options with up/down arrows

### Expected behaviour

With NVDA active, if i focus on a MultiActionButton, I can open the options list with enter or down arrow and then navigate through the options with up/down arrows

### CodeSandbox or Storybook URL

https://carbon.sage.com/?path=/docs/multi-action-button--default-story

### Carbon Version

119.3.0

### Design Tokens Version

4.17.0

### What browsers are you seeing the problem on?

Chrome

### What Operating System are you seeing the problem on?

Windows

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by Parsium** [comment]
FE-6178

**Comment by gusch** [comment]
Same explanation than https://github.com/Sage/carbon/issues/6245.
It is a normal behaviour so i will close the ticket.
